src/screens/mainPage/MainPage.tsx

```diff
@@ -1,23 +1,18 @@
-import {useEffect, useState} from "react"
+import {useEffect, useContext} from "react"
 import CardList from "./components/cardList/CardList"
 import Togglers from "./components/togglers/Togglers"
 import ImageArea from "./components/imageArea/imageArea"
-import {I_Profile} from "../../types/types"
 import Api from "../../api/Api"
-
+import {ProfileContext} from "../../context/profile.context"
 
 const MainPage: React.FC = () => {
-    const [mode, setMode] = useState<string>("Relevant")
-    const [profiles, setProfiles] = useState<I_Profile[]>([])
-
-    console.log("actial data", profiles)
 
+const {profiles, setProfiles, mode} = useContext(ProfileContext)
 
-    const changeMode = (event: React.MouseEvent<HTMLButtonElement>, toggler:string) => {
-    setMode(toggler) 
-    }
+    console.log("actial data", profiles)
```

Comment:
If something works as expected, it is best practice to remove logging before merges. Not essential though...